features/promotion/managing_promotions/prevent_from_removing_products_that_are_used_in_promotion_rules.feature

```diff
@@ -0,0 +1,17 @@
+@managing_promotions
+Feature: Preventing from removing products that are used in promotion rules
+    In order to prevent unwanted behaviour
+    As an Administrator
+    I need to be prevented from removing products that are used in promotion rules
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "Mug" and "Cup" products
+        And I am logged in as an administrator
+
+    @ui @api
+    Scenario: Being prevented from removing a product that is used in a promotion rule of type contains product
```

Comment:
```suggestion
    Scenario: Being prevented from removing a product that is used by any promotion rule
```

Reply:
I'll just comment here, since there's multiple places with the same change.
Lets just use `is in use by a promotion rule` globally, instead of used/assigned/etc